ecosystem/sep-0040.md

```diff
@@ -0,0 +1,160 @@
+## Preamble
+
+```
+SEP: To Be Assigned
+Title: Non-Fungible Token Standard
+Author: George Kudrayvtsev <george@stellar.org>
+Track: Standard
+Status: Draft
+Created: 2021-01-31
+Updated: 2021-01-31
+Version: 0.1.0
+Discussion: TBD
+```
+
+## Simple Summary
+This SEP defines a standard interface to describe non-fungible tokens (NFTs) on the Stellar network, including a distribution mechanism and metadata fields. It draws inspiration from the [EIP-721 standard](https://eips.ethereum.org/EIPS/eip-721).
+
+
+## Motivation
+A standard interface for NFTs allows ecosystem applications (such as wallets or marketplaces) to create, view, or otherwise work with any NFT on Stellar.
+
+
+## Abstract
+Non-fungible tokens (NFTs) can describe many types of assets, from digital collectibles to physical items. We should also define a standard issuance model to explain how NFTs can and should be issued on the network. In order to maintain compatibility across the Stellar ecosystem and beyond, we need a standard way to describe and define them that is agnostic of what the NFT actually describes.
+
+
+## Specification
+We begin with the [issuance model](#issuance-model) and then cover the NFT [metadata specification](#nft-metadata-specification).
```

Comment:
It can be helpful for readers if the protocol describes the protocol and doesn't need to tell a story about how the protocol comes to be, so we probably don't need the `We begin` prose. @tomerweller raised a similar comment to this on another protocol we wrote recently with some reasoning that is maybe more elegant than I'm suggesting, so if he chimes in here he might have some thing more thoughtful to say about it. For example:
```suggestion
The protocol defines an [issuance model](#issuance-model) that defines how NFT assets are issued on Stellar, and a [metadata specification](#nft-metadata-specification) that defines how data about the NFT is captured in a way that Stellar guarantees is immutable.
```

I also think that this specification introduction is maybe superfluous and you could drop it if the abstract say substantially the same thing.

Reply:
Yep, I think this is a good point, I just wanted it to flow a little more naturally (be "prose-y").